dlib/cmake_utils/cmake_find_blas.txt

```diff
@@ -346,3 +348,22 @@ if (NOT lapack_found)
     endif()
 endif()
 
+# If using lapack, determine whether to mangle functions
+if (lapack_found)
+   include(CheckFunctionExists)
+   include(CheckFortranFunctionExists)
+   set(CMAKE_REQUIRED_LIBRARIES ${LAPACK_LIBRARIES})
```

Comment:
I just tested this and it doesn't work unless `LAPACK_LIBRARIES` is all lowercase since cmake is case sensitive.  For instance, I've got the Intel MLK installed and this bit of code doesn't find it because all the other variables in this file use the lowercase version of this variable name.  Can it be changed to lowercase or is there a reason you did it this way?

Reply:
Apologies - this was just an oversight. I've now changed the variable to the correct capitalization